.github/workflows/dependent-issues.yml

```diff
@@ -2,8 +2,18 @@ name: Dependent Issues
 
 on:
   issues:
+    types:
+      - opened
+      - edited
+      - reopened
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - reopened
+      - synchronize
   schedule:
-    - cron: '0/30 * * * *' # schedule daily check
+    - cron: '0/60 0 * * *' # schedule hourly check
```

Comment:
I would suggest making it once a day. It's not required for running it every hour. Previously it didn't run on PR event so we have to do so. But here it is not required. Change it to once a day.

Reply:
Ahh you're right. Can you suggest what is the right crontab for once daily?